clients/cli/src/selector.rs

```diff
@@ -0,0 +1,142 @@
+use crate::utils::get_by_path;
+use crate::CliError;
+use dialoguer::theme::ColorfulTheme;
+use dialoguer::{FuzzySelect, Input};
+use lockbook_core::Filter::{DocumentsOnly, FoldersOnly};
+use lockbook_core::{
+    Core, CreateFileAtPathError, CreateFileError, DecryptedFileMetadata, FileType,
+    GetFileByPathError, Uuid,
+};
+use lockbook_core::{Error as LbError, GetFileByIdError};
```

Comment:
The cli imports are organized to begin with, so let's tidy these up. Also, wouldn't import Filter variants.

Reply:
de788ac0